lib/LaTeXML/Util/Pathname.pm

```diff
@@ -410,16 +410,24 @@ sub pathname_kpsewhich {
   if ($kpse_toolchain) {
     push(@candidates, $kpse_toolchain); }
   if ($kpsewhich && open(my $resfh, '-|', $kpsewhich, @candidates)) {
-    my $result = <$resfh>;     # we only need the first line
+    my $ignore = 0;
+    my $result;
+    while (<$resfh>) {
+      chomp;
+      if (/^=+$/) {    # ignore MikTeX maintenance info between ==== lines
+        $ignore = !$ignore;
+        next; }
+      if (!$ignore && $_) {
+        $result = $_;    # we only need the first non-empty result
```

Comment:
General note: the less you use the implicit `$_` (or other Perl implicits) in latexml code, the better.

It's very easy to have strange behavior slip through unnoticed in such cases.

Reply:
> General note: the less you use the implicit `$_` (or other Perl implicits) in latexml code, the better.

All very true. I kind of feel embarrassed now. My money is on that `/^=+$/` test – I don't see what else could interfere with `$_`.